src/components/parts/Radio.tsx

```diff
@@ -1,16 +1,33 @@
+import classNames from 'classnames/bind';
 import { InputHTMLAttributes, forwardRef } from 'react';
-import Input from './Input';
 import Label from './Label';
 
+import styles from './Radio.module.scss';
+
+const cn = classNames.bind(styles);
+
 interface RadioProps extends InputHTMLAttributes<HTMLInputElement> {
   isError?: boolean;
+  isChecked?: boolean;
 }
 
-export default forwardRef<HTMLInputElement, RadioProps>(function Radio({ id, value, checked, isError, ...rest }, ref) {
+export default forwardRef<HTMLInputElement, RadioProps>(function Radio(
+  { id, isChecked, value, isError, onChange, children, ...rest },
+  ref,
+) {
   return (
     <Label htmlFor={id} sizeVariant='md'>
-      <Input ref={ref} id={id} type='radio' value={value} checked={checked} isError={isError} {...rest} />
-      {value}
+      {/* <Input ref={ref} id={id} type='radio' checked={checked} isError={isError} onChange={onChange} {...rest} /> */}
+      <input
+        className={cn('radio', { red: isError })}
+        ref={ref}
+        id={id}
+        type='radio'
+        checked={isChecked || undefined}
```

Comment:
checked에 undefined 형식 넣어도 되나요?
```suggestion
        checked={isChecked ?? false}
```

Reply:
둘다 똑같이 작용하긴 해요